cypress/e2e/marc/marc-authority/edit-linked-authority-record/marc-authority-edit-linked-without-010-field-when-001-field-value-set-to-0.cy.js

```diff
@@ -0,0 +1,148 @@
+      numOfRecords: 1,
+    },
+  ];
+
+  before('Create test data', () => {
+    cy.loginAsAdmin({
+      path: TopMenu.dataImportPath,
+      waiter: DataImport.waitLoading,
+    }).then(() => {
+      marcFiles.forEach((marcFile) => {
+        cy.visit(TopMenu.dataImportPath);
+        DataImport.verifyUploadState();
+        DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+        JobProfiles.waitFileIsUploaded();
+        JobProfiles.waitLoadingList();
+        JobProfiles.search(marcFile.jobProfileToRun);
+        JobProfiles.runImportFile();
+        JobProfiles.waitFileIsImported(marcFile.fileName);
+        Logs.checkStatusOfJobProfile(JOB_STATUS_NAMES.COMPLETED);
+        Logs.openFileDetails(marcFile.fileName);
+        Logs.getCreatedItemsID().then((link) => {
+          createdRecordIDs.push(link.split('/')[5]);
+        });
+      });
+    });
+
+    cy.createTempUser([
+      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordEdit.gui,
+      Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
+      Permissions.inventoryAll.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+      Permissions.uiQuickMarcQuickMarcAuthorityLinkUnlink.gui,
+    ]).then((createdUserProperties) => {
+      testData.userProperties = createdUserProperties;
+
+      cy.visit(TopMenu.inventoryPath).then(() => {
+        InventoryInstances.searchByTitle(createdRecordIDs[0]);
+        InventoryInstances.selectInstance();
+        InventoryInstance.editMarcBibliographicRecord();
+        InventoryInstance.verifyAndClickLinkIcon(testData.tag240);
+        MarcAuthorities.switchToSearch();
+        InventoryInstance.verifySelectMarcAuthorityModal();
+        InventoryInstance.verifySearchOptions();
+        InventoryInstance.searchResults(testData.authority100FieldValue);
+        MarcAuthorities.checkFieldAndContentExistence(
+          testData.tag100,
+          `$a ${testData.authority100FieldValue}`,
+        );
+        InventoryInstance.clickLinkButton();
+        QuickMarcEditor.verifyAfterLinkingAuthority(testData.tag240);
+        QuickMarcEditor.pressSaveAndClose();
+        QuickMarcEditor.checkAfterSaveAndClose();
+
+        cy.login(testData.userProperties.username, testData.userProperties.password, {
+          path: TopMenu.marcAuthorities,
+          waiter: MarcAuthorities.waitLoading,
+        });
+      });
+    });
+  });
+
+  after('Delete test data', () => {
+    Users.deleteViaApi(testData.userProperties.userId);
+    InventoryInstance.deleteInstanceViaApi(createdRecordIDs[0]);
+    MarcAuthority.deleteViaAPI(createdRecordIDs[1]);
+  });
+
+  it(
+    'C375139 Edit any field in linked "MARC authority" record without "010" field when "001" = "$0" (spitfire) (TaaS)',
+    { tags: ['extendedPath', 'spitfire'] },
+    () => {
+      MarcAuthorities.searchAndVerify(testData.searchOption, marcFiles[1].authorityHeading);
+
```

Comment:
step 2 - Not all verifications were covered
![Screenshot 2023-12-22 163347](https://github.com/folio-org/stripes-testing/assets/114989418/3245d44a-080d-40b2-86b5-34a146acf1d1)